JsonLd/Resource.php

```diff
@@ -272,6 +312,90 @@ public function getRouteCollection()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getCollectionRoute()
+    {
+        if (!$this->collectionRoute) {
+            $this->getRouteCollection();
```

Comment:
Only the `routeCollection` attribute seems to be populated by `getRouteCollection` method, not the `collectionRoute` attribute. Why there's these 2 vars with a too much similar name ?